src/main/java/frc/robot/subsystems/ShooterHoodSubsystem.java

```diff
@@ -103,7 +109,8 @@ private boolean hasReliableZero() {
   }
 
   private boolean isFullyDeployed() {
-    return Math.abs(getHoodPos() - currSetpoint) < MoPrefs.getShooterHoodPositionTolerance();
+    return MoPrefs.getShooterHoodSetpoint() - MoPrefs.getShooterHoodPositionTolerance() < getHoodPos()
```

Comment:
I was having a lot of trouble with the it, so I messed around with the logic.  Eventually, it turned out that getHoodPos() wasn't actually polling the encoder.  I just didn't change it back because we moved on to tuning the shooter flywheel PID.  I'll fix it once I finish my homework.

Reply:
Another case of needing better understanding before hacking.
I get that if you aren't familiar with the standard tolerance pattern that it may have seemed odd, but now it seems odd and difficult to read for the rest of us.

As a general rule, any time you have to work to understand a bit of code, that's a screaming flag that it deserves a comment block to explain what's going on and why.  This is doubly true if you need to change something to work around a bug.  When the real bug gets fixed, it needs to be easy to find any workarounds that will no longer behave as expected and fix them too.